file-browser/src/Meta.js

```diff
@@ -6,8 +6,9 @@ import ContextHeader from "./components/ContextHeader";
 import ListTab from "./components/ListTab";
 import ListView from "./components/ListView";
 import GenericFileItem from "./components/GenericFileItem";
+import SelectContainer from "./packages/selector-container/SelectContainer";
 
-export default class Meta extends React.Component {
+class Meta extends React.Component {
     static propTypes = {
         store$: PropTypes.any
```

Comment:
@jiamingguo It is a `rxjs` <= reactive js idiom, it stands for stream. It means the object is a publisher/Observable that someone can subscribe to.